src/Confluent.Kafka/Impl/NativeMethods/NativeMethods.cs

```diff
@@ -408,23 +408,12 @@ internal class NativeMethods
         [DllImport(DllName, CallingConvention = CallingConvention.Cdecl)]
         internal static extern ErrorCode rd_kafka_position(
                 IntPtr rk, IntPtr partitions);
-
-        // note: producev signature is rd_kafka_producev(rk, ...)
-        // we are keeping things simple with one binding for now, but it 
-        // will be worth benchmarking the overload with no timestamp, opaque,
-        // partition, etc
+        
```

Comment:
this file is not like the other three... they should all match.

worth keeping this comment i think, but get rid of the line 
```
// note: producev signature is rd_kafka_producev(rk, ...)
```

and prefix `note:` on the next line.